### ISSUE TITLE
Add formater/linter

### ISSUE THREAD

**Issue by oskardudycz** [oskardudycz/EventSourcing.JVM#6]
Add formater and/or linter, e.g. Checkstyle: https://www.youtube.com/watch?v=RAC_VRj2bcM

**Comment by jkornata** [comment]
@oskardudycz If you are fine with https://github.com/google/google-java-format I can make a PR with a gradle plugin verifying it.

**Comment by oskardudycz** [comment]
@jkornata, yup, I'm fine. It'd be great if you could send such a contribution! It'd be more than welcome 🙂

**Comment by timmycam** [comment]
Hey didn't see any movement on this one so raised a quick PR, not sure that some of the rules are all that great by default, but can have a look and see if there is a common workaround for some of them? eg. the lamdas.
